tools/pyclient/src/molgenis_emx2_pyclient/client.py

```diff
@@ -91,3 +126,120 @@ def list_databases(self):
         database_names = [db['name'] for db in databases]
 
         return database_names
+    
+    @staticmethod
+    def _prep_data_or_file(file: str = None, data: list = None) -> pd.DataFrame:
+        """Prepares the data from memory or loaded from disk for addition or deletion action.
+
+        :param file: path to the file to be prepared
+        :type file: str
+        :param data: data to be prepared
+        :type data: list
+
+        :returns: prepared data in dataframe format
+        :rtype: pd.DataFrame
+        """
+        if not bool(file) and not bool(data):
+            print('No data to import. Specify a file location or a dataset.')
+        
+        if bool(file):
+            return utils.read_file(file=file)
+          
+        if bool(data):
+            return utils.to_csv(data=data)
+    
+    def add(self, schema: str, table: str, file: str = None, data: list = None):
+        """Imports or updates records in a table of a named schema.
+        
+        :param schema: name of a schema
+        :type schema: str
+        :param table: the name of the table
+        :type table: str
+        :param file: location of the file containing records to import or update
+        :type file: str
+        :param data: a dataset containing records to import or update (list of dictionaries)
+        :type data: list
+        
+        :returns: status message or response
+        :rtype: str
+        """
+        import_data = self._prep_data_or_file(file=file, data=data)
+
+        if schema not in self.schemas:
```

Comment:
as said below I would probably myself prefer to have a way to set a default schema so I can make my scripts more easily schema independent. Doesn't need to be done in this PR (is there a place were ideas are being managed?)

Reply:
This has been implemented